style.css

```diff
@@ -0,0 +1,141 @@
+@font-face {
+	font-family: 'a_LCDNova';
+	src: url('fonts/3574.eot');
+	src: local('☺'), url('fonts/3574.woff') format('woff'), url('fonts/3574.ttf') format('truetype'), url('fonts/3574.svg') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+}
+
+body {
+  background-color: rgb(193, 188, 183);
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  align-content: center;
+  justify-content: center;
+  overflow: auto;
+  font-family: "a_LCDNova", sans-serif;
+}
+
+.calc {
+  width: 300px;
+  height: 464px;
+  background: linear-gradient(rgb(92, 89, 84), rgb(82, 79, 74)) rgb(92, 89, 84);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 6px;
+  box-shadow: 0 8px 5px rgba(29, 24, 11, 0.8), 2px 3px 1px 1px rgba(0, 1, 6, 0.3);
+}
+
+.calc-display-input {
+  background-color: rgb(233, 234, 219);
+  margin: 20px 20px 0;
+  width: 260px;
+  height: 80px;
+  text-align: right;
+  font-size: 4.5rem;
+  font-family: "a_LCDNova", sans-serif;
+}
+
+button {
+  width: 53px;
+  height: 53px;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  text-transform: uppercase;
+  font-size: 3rem;
+  border-radius: 5px;
+  background-color: rgb(155, 152, 143);
+  cursor: pointer;
+}
+
+.calc__button {
+  margin: 20px;
+  display: grid;
+  gap: 15px;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(5, 1fr);
+}
+
+.button-zero {
+  grid-column: 1/3;
+  width: auto;
+}
+
+.button-equals {
+  grid-row: 4/6;
+  grid-column: 4/5;
+  height: auto;
+}
+
+.operation-style {
+  color: rgb(209, 209, 217);
+  text-decoration: none;
+  text-shadow: 0 -1px 2px rgba(0, 0, 0, 0.2);
+  outline: none;
+  background: linear-gradient(rgb(118, 111, 103), rgb(59, 55, 52)) rgb(118, 111, 103);
+  box-shadow: 0 1px rgba(255, 255, 255, 0.2) inset, 0 3px 5px rgba(0, 1, 6, 0.5), 0 0 1px 1px rgba(0, 1, 6, 0.2);
+  transition: 0.2s ease-in-out;
+}
+
+.operation-style:hover:not(:active) {
+  background: linear-gradient(rgb(138, 131, 123), rgb(49, 45, 42)) rgb(138, 131, 123);
+}
+
+.operation-style:active {
+  top: 1px;
+  background: linear-gradient(rgb(98, 91, 83), rgb(39, 35, 32)) rgb(98, 91, 83);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0.5) inset, 0 2px 3px rgba(0, 0, 0, 0.5) inset, 0 1px 1px rgba(255, 255, 255, 0.1);
+}
+
+.number-style {
+  color: rgb(209, 209, 217);
+  text-decoration: none;
+  text-shadow: 0 -1px 2px rgba(0, 0, 0, 0.2);
+  outline: none;
+  background: linear-gradient(rgb(173, 168, 163), rgb(114, 109, 102)) rgb(173, 168, 163);
+  box-shadow: 0 1px rgba(255, 255, 255, 0.2) inset, 0 3px 5px rgba(0, 1, 6, 0.5), 0 0 1px 1px rgba(0, 1, 6, 0.2);
+  transition: 0.2s ease-in-out;
```

Comment:
You repeat exactly the same style as for `.operation-style.` And also repeat it below for `.clear`. Consider some refactoring for this part, so that this style appears only once in your code.

Reply:
DONE!